### PR TITLE
Comment monitoring imports in the sample memcached-operator

### PR DESCRIPTION
**Pull request by assafad**
Signed-off-by: assafad <aadmi@redhat.com>

**Description of the change:**
This PR changes the monitoring imports in the sample memcached-operator code to be comments by default. Users who want to use custom metrics for the operator, will uncomment these lines. Another separate PR will add a custom metric tutorial for the operator-SDK documentation, and there the users will be instructed to uncomment these lines. 

**Motivation for the change:**
https://github.com/operator-framework/operator-sdk/issues/6008

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

**Comment by assafad** [comment]
Hi @camilamacedo86, @everettraven, could you please have a look?

File: hack/generate/samples/internal/go/memcached-with-webhooks/memcached_with_webhooks.go [hunk 1]
```diff
@@ -532,7 +532,7 @@ func (mh *Memcached) customizingMain() {
 	// Add metrics registry
 	err = kbutil.InsertCode(mainPath,
 		"utilruntime.Must(cachev1alpha1.AddToScheme(scheme))",
-		"\n\nmonitoring.RegisterMetrics()")
+		"\n\n//monitoring.RegisterMetrics()")
```
Comment (camilamacedo86) [inline hack/generate/samples/internal/go/memcached-with-webhooks/memcached_with_webhooks.go:535]: Could we use a const here to make eaiser to maintain ( see the fragments used in another places )
also, note that IDEA provide plugins to work with strings which make that eaiser

Also, could we please add a comment explaining why the code is commented and when it is useful
By latest we need to comment the import as well.

Comment (assafad) [reply hack/generate/samples/internal/go/memcached-with-webhooks/memcached_with_webhooks.go]: Done

**Review by everettraven — APPROVED** [review]
I think this looks good.

One question, is the tutorial for enabling custom metrics still in the works? If so, when that is finished it would be nice to include a link in the comments that have to do with enabling custom metrics to point to the doc.

**Comment by assafad** [comment]
> I think this looks good.
> 
> One question, is the tutorial for enabling custom metrics still in the works? If so, when that is finished it would be nice to include a link in the comments that have to do with enabling custom metrics to point to the doc.

@everettraven yes, sounds good

File: hack/generate/samples/internal/go/memcached-with-webhooks/memcached_with_webhooks.go [hunk 2]
```diff
@@ -754,7 +754,8 @@ const controllerImportsFragment = `
 `
 
 const controllerMonitoringImportFragment = `
-	"github.com/example/memcached-operator/monitoring"
+	// To enable operator custom metrics, uncomment all sections with 'monitoring'
```
Comment (camilamacedo86) [inline hack/generate/samples/internal/go/memcached-with-webhooks/memcached_with_webhooks.go:757]:
```suggestion
	// This sample has an example of how to create and work with custom metrics. The code is commented
	// in order to not break the basic tutorial to show authors how to build an operator. 
	// To enable operator custom metrics, uncomment all sections with 'monitoring'
```

**Review by camilamacedo86 — COMMENTED** [review]
Hi @assafad, 

It shows fine, thx for the contribution. 🥇 

Just a nit suggesting about the comment see:  https://github.com/operator-framework/operator-sdk/pull/6018/files#r964329590

Also, you need to rebase it with master before move forward.

File: hack/generate/samples/internal/go/memcached-with-webhooks/memcached_with_webhooks.go (hunk 2 above)
Comment (camilamacedo86) [reply hack/generate/samples/internal/go/memcached-with-webhooks/memcached_with_webhooks.go:757]: Could we try to give a little bit more context here @assafad, wdyt?

Comment (assafad) [reply hack/generate/samples/internal/go/memcached-with-webhooks/memcached_with_webhooks.go]: Done

**Review by camilamacedo86 — APPROVED** [review]
/lgtm

**Review by everettraven — APPROVED** [review]
/lgtm

**Pull request merged**